resources/xml/fedramp_values.xml

```diff
@@ -524,4 +524,25 @@
       </allowed-values>
    </value-set>
 
+   <value-set name="role-type">
```

Comment:
this is a good addition for any follow on work for #32 as well as extending the domain to support #50

Reply:
I ironically did not even notice this in the merge conflict cleanup! So Brian has been adding some and I missed it or he merged one of our PRs, I guess we should discuss this then later today.

Reply:
I guess I am getting pretty forgetful, it was merged upstream in GSA/fedramp-automation#68 long enough I had since forgotten about the PR.